src/models/announcement.cr

```diff
@@ -1,8 +1,11 @@
 require "granite_orm/adapter/pg"
 require "markdown"
 require "autolink"
+require "../helpers/*"
 
 class Announcement < Granite::ORM::Base
+  include Helpers::TimeAgoHelper
```

Comment:
Let's use helpers in controllers. There are plenty of inclusions in `ApplicationController`. So you just would do `time_ago_in_words announcement.create_at.not_nil!` in the view.

Reply:
I was trying to avoid this kind of logic going into views, but the project is still small enough to don't care about presenters :smile:

Reply:
I always have doubts where to put that or another logic, but helper sounds to me like presentation related stuff 😄